api/handler/v1/governance.go

```diff
@@ -19,11 +19,18 @@ func ProcessProposals(proposalsRes string, v1 bool) ([]byte, error) {
 		return []byte{}, err
 	}
 
+	var filteredProposals []blockchain.V1GovernanceProposal
+	for _, proposal := range jsonProposalRes.Proposals {
```

Comment:
Probably better to range without creating a copy of the value (proposal). Something like:

```
for index := range jsonProposalRes.Proposals {
if jsonProposalRes[i].Status .....
```